server/game/cards/02-AoA/LittleRapscal.js

```diff
@@ -5,10 +5,11 @@ class LittleRapscal extends Card {
         this.persistentEffect({
             targetController: 'any',
             match: card => card.type === 'creature' && ability.actions.fight().canAffect(card, this.game.getFrameworkContext(card.controller)),
-            effect: [
+            effect: ability.effects.mustFightIfAble()
+            /* effect: [
```

Comment:
If you're happy this replaces what was there before please remove the commented code